site/docs/hosting/heroku.md

```diff
@@ -44,11 +44,11 @@ Our folder structure should now look like this:
 
 After that, open `tsconfig.json` and change it to use this configuration:
 
-```json
+```json{4}
```

Comment:
What is this even doing?

Reply:
It's highlighting those lines.
Well i just thought it was worth a shot , but yes it's a very small change , may look inappropriate also.
No worries.

Reply:
That's cool! I didn't know the highlight feature existed.

![Screenshot Hosting Heroku grammY](https://user-images.githubusercontent.com/74030149/180135084-13493349-ff5e-4874-8a74-0556c9295a64.png)

Reply:
Me neither, I like this a lot! I didn't know VuePress was that cool 😄

Reply:
lol i thought it's a obvious thing so i didn't do much to explain the problem 
my bad!!!

the whole point was to catch the eye of the user instantly that which part they need to change instead of matching the whole config and maybe making mistakes in between 

but i am glad everyone likes it!!!